### PR TITLE
Add VS2017 native SDK to bundle.

### PR DESCRIPTION
**Pull request by barnson**
wixtoolset/issues#5538

### CODE REVIEW

**Comment by barnson** [comment]
#triage

File: src/Setup/Bundle/PackageGroups.wxs [hunk 1]
```diff
@@ -64,6 +64,12 @@
                 <MsiProperty Name='INSTALLFOLDER' Value='[InstallFolder]' />
             </MsiPackage>
             <?endif?>
+
+            <?ifdef VS2017Available ?>
```
Comment (rseanhall) [inline src/Setup/Bundle/PackageGroups.wxs]: `VS2017Available` is not defined anywhere

Comment (barnson) [inline src/Setup/Bundle/PackageGroups.wxs]: tools\WixBuild.props

Comment (rseanhall) [inline src/Setup/Bundle/PackageGroups.wxs]: MSBuild property != WiX variable, I expected it at https://github.com/wixtoolset/wix3/blob/develop/tools/WixBuild.wixproj.props. The libs didn't get installed on my machine.

Comment (barnson) [inline src/Setup/Bundle/PackageGroups.wxs]: D'oh.

Comment (barnson) [inline src/Setup/Bundle/PackageGroups.wxs]: All better. Tested, even.

**Pull request closed without merging**